WalletWasabi.Gui/Controls/WalletExplorer/CoinListView.xaml

```diff
@@ -54,7 +54,7 @@
         <StackPanel Spacing="10" DockPanel.Dock="Bottom" Orientation="Horizontal" IsVisible="{Binding IsAnyCoinSelected}">
           <TextBlock Text = "|" />
           <TextBlock Text = "Selected Amount:" />
-          <TextBlock Text = "{Binding TotalAmount, ConverterParameter=8, Converter={StaticResource LurkingWifeModeStringConverter}}" />
+          <TextBlock Foreground="YellowGreen" Text = "{Binding TotalAmount, ConverterParameter=8, Converter={StaticResource LurkingWifeModeStringConverter}}" />
```

Comment:
What does this do?

Reply:
Makes the selected btc amount the same green colour as other values, it's hard to notice when you first select coins with the other white/grey text.